service/start.go

```diff
@@ -126,25 +145,41 @@ func (svc *service) startAllExistingAppsWalletSubscriptions(ctx context.Context,
 
 	for _, app := range apps {
 		go func(app db.App) {
-			err := svc.startAppWalletSubscription(ctx, relay, *app.WalletPubkey, "")
+
+			// get nip47 event info for this app wallet key
+			nip47InfoEvent, err := svc.GetNip47Service().GetNip47Info(ctx, relay, *app.WalletPubkey)
```

Comment:
what is the benefit of fetching the info first?

Reply:
Do we really need to do this? couldn't it cause more likelyhood of errors (and if an error occurs, the subscription is not created)

Reply:
You're right, we can do the fetching only when needed on the `deleteAppConsumer`.
We need to fetch the event since for deleting it we need it's `ID`